C7FIT/User.swift

```diff
@@ -13,24 +13,51 @@ import Firebase
     A model representation of a User
  */
 struct User {
-//    let uid: String
     let email: String
-//    let name: String?
-//    let weight: Int?
-//    let height: Int?
-//    let bmi: Double?
-//    let mileTime: String?
-
-    init(authData: FIRUser) {
-//        uid = authData.uid
-        email = authData.email!
-    }
+    let photoURL: String?
+    let name: String?
+    let bio: String?
+    let weight: String?
```

Comment:
If your deployment target is iOS 10 you should consider using NSMeasurment for values that incorporate unit information. Really nice new API.

https://developer.apple.com/reference/foundation/nsmeasurement